packages/react-components/src/styles/components.ts

```diff
@@ -118,4 +118,10 @@ export default css`
     z-index: 1002;
     max-width: 300px;
   }
+
+  .content-container {
+    width: 100%;
+    max-width: 1088px;
```

Comment:
Too small. This breaks democracy and things like tips. For optimal viewing pleasure 1792px. So this should only apply on exceptionally large screens.

Additionally, all the apps _should_ be wrapped in `<main>...</main>` so it can possibly apply at that level.

So overall -

- probably don't need to apply it anywhere explicitly, just add to main (unless it doesn't work) with center
- keep it to Jaco's screen default of 1792px (1800px screen), anything less than that and we start dropping columns

Reply:
![image](https://user-images.githubusercontent.com/1424473/96150925-28e3b280-0f0b-11eb-90a6-4282d794ed76.png)

Reply:
If add .content-container class to main, then tabs will be broken.
![image](https://user-images.githubusercontent.com/25020300/96225749-d5687780-0f91-11eb-9739-b8d56bf47d5f.png)